.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "8"
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+install:
+  - npm version
+  - npm install
+script:
+  - npm run build
+  - npm test
```

Comment:
Can't we add `npm run e2e` ?

Reply:
e2e has chrome dependency. Will try to get that working in travis ci.